src/Microsoft.DotNet.ProjectModel/ProjectSystem/DependencyDescription.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Extensions.ProjectModel.ProjectSystem
```

Comment:
Most of the other files under this repo are run `Microsoft.Extension.*` namespace.